portability-transfer/src/main/java/org/datatransferproject/transfer/logging/EncryptingLayout.java

```diff
@@ -0,0 +1,36 @@
+package org.datatransferproject.transfer.logging;
+
+import com.google.common.base.Preconditions;
+import java.util.UUID;
+import org.apache.log4j.Layout;
+import org.apache.log4j.helpers.ISO8601DateFormat;
+import org.apache.log4j.spi.LoggingEvent;
+
+public class EncryptingLayout extends Layout {
+
+  static UUID jobId;
+
+  public static void setJobId(UUID inputJobId) {
+    Preconditions.checkState(jobId == null, "jobId has already been set");
+    jobId = inputJobId;
+  }
+
+  @Override
+  public String format(LoggingEvent event) {
+    // NB: copied from SimpleLayout.format()
+    return String.format("[%s] [%s]: %s - %s%s",
```

Comment:
Cna we either change to read this from the config file or leave a TODO to do that?